Logger.md

```diff
@@ -0,0 +1,107 @@
+# Logger
+
+`Logger` is an interface that helps decouple client classes from concrete implementations of a logging system.
+
+## Declaration
+
+```swift
+public enum LogLevel {
+    case info
+    case warning
+    case error
+}
+
+public protocol Logger {
+  func log(_ level: LogLevel, tag: String, message: String)
+  func print(tag: String, message: String)
```

Comment:
question: why do we call it `print` if this is a `.info` log level? why not keeping the naming conventions you use with `warning` and `error`? 

I think we should call it `func info(...)` or rename the `.info` to `.print` to keep consistency.

Reply:
Initially, we were mirroring Bugfender's mobile library API. Changed.

Reply:
`print()` function is the name convention in all the languages for the default implementation but if we want to against...

Reply:
I do agree on this. My comment here was in an effort to keep consistency between the three enum options and the three methods.

We can call the enum `.print` or even`.default`, or just leave it as `.info` if we think is better.

Reply:
- Reverted method name to be print.
- I prefer .info, .warning, .error.

Reply:
great then.